docs/content/connector-sink.md

```diff
@@ -111,6 +113,7 @@ In your Maven project's `pom.xml` file, add the Flink connector as a dependency
 | sink.properties.max_filter_ratio  | No           | 0                 | The maximum error tolerance of the Stream Load. It's the maximum percentage of data records that can be filtered out due to inadequate data quality. Valid values: `0` to `1`. Default value: `0`. See [Stream Load](https://docs.starrocks.io/en-us/latest/sql-reference/sql-statements/data-manipulation/STREAM%20LOAD) for details.                                                                                                                                                                                                                                                                                                                                                                      |
 | sink.properties.strict_mode       | No           | false             | Specifies whether to enable the strict mode for Stream Load. It affects the loading behavior when there are unqualified rows, such as inconsistent column values. Valid values: `true` and `false`. Default value: `false`. See [Stream Load](https://docs.starrocks.io/en-us/latest/sql-reference/sql-statements/data-manipulation/STREAM%20LOAD) for details.                                                                                                                                                                                                                                                                                                                                                                      |
+| sink.properties.compression       | No           | NONE              | The compression algorithm used for Stream Load. Currently only support json format. Valid values: `lz4_frame`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
The compression algorithm used for Stream Load. Currently, compression is only supported for the JSON format. Valid values: `NONE` (compression will not be used) and `lz4_frame`.

Reply:
Refine the description. `NONE` means not set the option, but it's not a valid value.